### PR TITLE
Changes to readme.md

### PR DESCRIPTION
**Pull request by blacng**
Hello @alozano-77 

Thanks for contributing lesson notes for A. Cantrill's SAA course. I have made some changes to the readme and will like to contribute to the repository. 

Best regards,
Seun

### CODE REVIEW

File: 06-S3/notes.md [hunk 1]
```diff
@@ -0,0 +1,672 @@
...
+By default once an object is uploaded to S3, it is sent as a single stream
+of data. If a stream fails, the whole upload fails. This requires a full
+restart of the data transfer.
+
+Single PUT upload up to 5GB
+
+#### Multipart Upload
+
+Data is broken up into smaller parts.
+The minimum daa size is 100 MB for multipart.
```
Comment (etolbakov) [inline 06-S3/notes.md:142]: [typo] data

File: 06-S3/notes.md [hunk 2]
```diff
@@ -0,0 +1,672 @@
...
+
+Resource policies can allow or deny anonymous principals. This is explicitly
+declared in the bucket policy itself.
+
+Each bucket can only have one policy, but it can have multiple statements.
+
+#### ACL's (Legacy)
+
+A way to apply a subresource to objects and buckets.
+These are legacy and AWS does not recomment their use.
```
Comment (etolbakov) [inline 06-S3/notes.md:33]: [typo] recommend

**Review by alozano-77 — APPROVED** [review]
What an update! I apologize for the delay, but thank you so much for helping the community!

**Review by alozano-77 — APPROVED** [review]

**Review by alozano-77 — APPROVED** [review]

**Comment by alozano-77** [comment]
Thank you @etolbakov for reviewing. I hope I processed your reviews properly. I apologize in advance if I did not, but thanks for the help!

**Comment by etolbakov** [comment]
My pleasure @alozano-77
Thank you very much for this awesome material!!!

**Pull request merged**